tests/fixtures/ssl/Makefile

```diff
@@ -0,0 +1,12 @@
+CA_PASSWORD?=strong_password
+PASSWORD?=strong_password
+OUTPUT_FOLDER=.
```

Comment:
Is this safe when running make from another directory, e.g. `make -C tests/fixtures/ssl ssl_keys`  ?
Does "." mean $PWD or the directory the makefile resides in?

Reply:
It was safe with make, what wasn't safe was calling directly `./tests/fixtures/ssl/create_keys.sh`. Removed here as not needed and fixed there.